neuromation/cli/job.py

```diff
@@ -59,6 +59,9 @@
 
 log = logging.getLogger(__name__)
 
+ENV_VAR_STORAGE_ROOT_MOUNTPOINT = "NP_STORAGE_ROOT_MOUNTPOINT"
```

Comment:
```suggestion
ENV_VAR_STORAGE_ROOT_MOUNTPOINT = "NP_STORAGE_ROOT"
```
or `NP_STORAGE_PATH` to make it shorter.

Reply:
ok